octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -799,26 +800,17 @@ async def order(self, order: cryptofeed_types.OrderInfo, receipt_timestamp: floa
         :param order the order object defined in cryptofeed.types.?
         :param receipt_timestamp: received timestamp
         """
-        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, [{
-            trading_enums.ExchangeConstantsOrderColumns.TYPE.value: self._parse_order_type(order.type),
-            trading_enums.ExchangeConstantsOrderColumns.STATUS.value: self._parse_order_status(order.status),
```

Comment:
Doesn't we need a dedicated parser for cryptofeed?

Reply:
havent tested it, but I included all logic so we dont need a separate parser anymore